UnityProject/Assets/Scripts/UI/Systems/MainHUD/PacketLossUI.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Linq;
+using Mirror;
+using UnityEngine;
+using UnityEngine.UI;
+using WebSocketSharp;
+using System.Net.NetworkInformation;
+using System.Threading;
+using Ping = System.Net.NetworkInformation.Ping;
```

Comment:
will fail code scanning

Reply:
which one?

Reply:
System.Net.NetworkInformation.Ping; isn't part of the white list

Reply:
how do i add this to the white list?